repo/packages/K/kafka/500/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "packagingVersion": "4.0",
+  "minDcosReleaseVersion": "1.10",
+  "upgradesFrom": [
+    "2.4.0-1.1.1"
+  ],
+  "downgradesTo": [
+    "2.4.0-1.1.1"
+  ],
+  "name": "kafka",
+  "version": "2.5.0-2.1.0",
+  "maintainer": "support@mesosphere.io",
+  "description": "Apache Kafka is used for building real-time data pipelines and streaming apps. It is horizontally scalable, fault-tolerant, wicked fast, and runs in production in thousands of companies.",
+  "selected": true,
+  "framework": true,
+  "tags": [
+    "message",
+    "broker",
+    "pubsub",
+    "kafka"
+  ],
+  "preInstallNotes": "Default configuration requires 3 agent nodes each with: 1.0 CPU, 2048 MB MEM and 1,5000 MB Disk.\n\nNote that Kafka requires a Zookeeper instance to be available before starting. It is a good practice to deploy a dedicated Zookeeper service, but you can always fall-back into using the DC/OS core Zookeeper instance instead.",
+  "postInstallNotes": "The DC/OS Apache Kafka service is being installed.\n\n\tDocumentation: https://docs.mesosphere.com/service-docs/kafka/\n\tIssues: https://docs.mesosphere.com/support/",
+  "postUninstallNotes": "The DC/OS Apache Kafka service is being uninstalled.\n\nFor DC/OS versions from 1.10 no further action is required. For older DC/OS versions follow the instructions at https://docs.mesosphere.com/service-docs/kafka/uninstall to remove any persistent state if required.",
+  "scm": "https://github.com/mesosphere/dcos-kafka-service",
+  "website": "https://docs.mesosphere.com/services/kafka/stub-universe",
```

Comment:
`s/stub-universe/2.5.0-2.1.0/g`

Note: Kafka 2.4.0-1.1.1 is being merged to directory `500`, as is this package which will create a merge conflict so this package must be merged to directory `600`.